backend/scripts/create_superuser.py

```
@@ -0,0 +1,34 @@
+    username = 'admin'
+    password = 'password123'
+    email = 'admin@shiningsmiles.edu'
+
+    if not User.objects.filter(username=username).exists():
+        user = User.objects.create_superuser(username=username, email=email, password=password)
+        print(f"Superuser '{username}' created successfully.")
+        
+        # Ensure profile exists and has Admin role
+        # Try/catch to handle if Profile is automatically created by signals
+        try:
+             profile, created = Profile.objects.get_or_create(user=user)
```

Ruff (0.14.8)

Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)